python/ray/tune/logger/tensorboardx.py

```diff
@@ -72,6 +72,24 @@ def on_result(self, result: Dict):
             ):
                 valid_result[full_attr] = value
 
+                # Must be image
+                if isinstance(value, np.ndarray) and value.ndim == 3:
+                    self._file_writer.add_image(
+                        full_attr,
+                        value,
+                        global_step=step,
+                    )
+                    continue
+
+                # Must be multi-image
```

Comment:
Dumb question: What if this is a single video (t, w, h, c)?

Reply:
Following the definition of [`add_video()`](https://tensorboardx.readthedocs.io/en/latest/tensorboard.html#tensorboardX.SummaryWriter.add_video) only 5-dimensional inputs are accepted for this function. 

Do we anywhere pass data in a lower dimensional array into this function?

Reply:
@sven1977 Do you see any cases where this setup of separating arrays by dimension could fall on our feet?